cookbooks/cdo-varnish/templates/default/default.vcl.erb

```diff
@@ -1,4 +1,5 @@
 import cookie;
+import header;
```

Comment:
using [libvmod-header](https://github.com/varnish/libvmod-header) for the `header.append` function, used to append a `Set-Cookie` header without removing any existing cookies generated by the application.